src/math/src/lib.cairo

```diff
@@ -233,3 +235,115 @@ impl U256BitRotate of BitRotate<u256> {
         remainder * pow(2, 256 - n) + quotient
     }
 }
+
+trait WrappingMath<T> {
+    fn wrapping_add(self: T, rhs: T) -> T;
+    fn wrapping_sub(self: T, rhs: T) -> T;
+    fn wrapping_mul(self: T, rhs: T) -> T;
+}
+
+impl U8WrappingMath of WrappingMath<u8> {
+    #[inline(always)]
+    fn wrapping_add(self: u8, rhs: u8) -> u8 {
+        u8_wrapping_add(self, rhs)
+    }
+
+    #[inline(always)]
+    fn wrapping_sub(self: u8, rhs: u8) -> u8 {
+        u8_wrapping_sub(self, rhs)
+    }
+
+    #[inline(always)]
+    fn wrapping_mul(self: u8, rhs: u8) -> u8 {
+        (u8_wide_mul(self, rhs) % 0x100_u16).try_into().unwrap()
```

Comment:
more efficient to use bitwise operations than using remainder of division

```suggestion
        (u8_wide_mul(self, rhs) & BoundedInt::<u8>::max().into()).try_into().unwrap()
```

Reply:
Makes total sense! Not sure why I hadn't noticed that.

Reply:
Done in [bb538bc](https://github.com/keep-starknet-strange/alexandria/pull/241/commits/bb538bcdb35c72cc8b75e03cde815f2f67ccd4e5).